packages/fcl/src/utils/web/default-config.js

```diff
@@ -1,21 +1,8 @@
-const isServerSide = () => typeof window === "undefined"
-
-const getSessionStorage = () => {
-  try {
-    const SESSION_STORAGE = {
-      can: !isServerSide(),
-      get: async key => JSON.parse(sessionStorage.getItem(key)),
-      put: async (key, value) => sessionStorage.setItem(key, JSON.stringify(value)),
-    }
-    return SESSION_STORAGE
-  } catch (error) {
-    return null
-  }
-}
```

Comment:
I scrapped the factory function here just because it feels simpler for the user to say `fcl.storage.default=LOCAL_STORAGE` than `fcl.storage.default=getLocalStorage()` in the external API.  It seems like a bit of an unnecessary abstraction since the object returned is always the same & there's no errors that can occur in the try-catch block anyways.

Happy to reconsider if there's any objections to this however.